.travis.yml

```diff
@@ -10,19 +10,20 @@ go:
   - 1.12.x
   - tip
 
-matrix:
-  allow_failures:
-    - go: tip
+before_install:
+  - go get -u github.com/kyoh86/richgo
+  - go get -u github.com/mitchellh/gox
+
+jobs:
   include:
     - stage: diff
       go: 1.12.x
-      script: diff -u <(echo -n) <(gofmt -d -s .)
-
-before_install: go get -u github.com/kyoh86/richgo
+      script: make fmt
+    - stage: test
+      script: make test
 
 script:
-  - richgo test -v ./...
-  - go build
+  - make
   - if [ -z $NOVET ]; then
       diff -u <(echo -n) <(go vet . 2>&1 | grep -vE 'ExampleCommand|bash_completions.*Fprint');
     fi
```

Comment:
Shouldn't this be added as another make target?

Reply:
yes, take note of "WIP" in the title of the PR.